Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -12,28 +12,32 @@ public enum ActorStates
 
 public class ActorPhysics : ActorComponent
 {
+	private new Transform transform;
```

Comment:
Why do this instead of naming a new variable? I get that we don't want to be doing getComponent all the time, but this seems hacky.